layout/theme.liquid

```diff
@@ -26,45 +26,46 @@
 <body id="{{ page_title | handle }}" class="template-{{ template | replace: '.', ' ' | truncatewords: 1, '' | handle }}" >
   <header id="header">
     <h1 class="u-visuallyhidden">{{ shop.name }}</h1>
-    <div class="container">
-      <nav class="grid-row font-uppercase">
-        <div class="grid-item item-s-4">
-          <a href="/about">About</a>
-        </div>
-        <div class="grid-item item-s-4 text-align-center">
-          <a href="/">Shop</a>
-        </div>
-        <div class="grid-item item-s-4 text-align-right">
-          <a href="/cart">{% if cart.item_count > 0%}<span>{{ cart.item_count }}</span> {% endif %}Cart</a>
-        </div>
-      </nav>
-    </div>
+    <nav id="header-nav" class="grid-row">
```

Comment:
Are there not `.container`s? I think even if we are going full fluid we should wrap in containers and just set that width to 100%. If things change we have the markup to edit quickly and it follows the rest of the markup we write.